packages/c/CMakePresets.json

```diff
@@ -50,6 +50,127 @@
       "name": "strict-ninja",
       "inherits": ["strict"],
       "generator": "Ninja"
+    },
+    {
+      "name": "release-static",
+      "binaryDir": "${sourceDir}/build/release-static",
+      "cacheVariables": {
+        "BUILD_SHARED_LIBS": "OFF",
+        "CMAKE_BUILD_TYPE": "Release",
+        "CMAKE_C_FLAGS": "-std=c99 -Wno-error -Wextra -Werror-implicit-function-declaration",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/build/release-static/tmp-install-dir",
+        "CMAKE_EXPORT_COMPILE_COMMANDS": "OFF",
+        "ATSDK_BUILD_TESTS": "OFF",
+        "ATSDK_MEMCHECK": "OFF",
+        "ENABLE_TESTING": "OFF",
+        "ENABLE_PROGRAMS": "OFF",
+        "ENABLE_CJSON_TEST": "OFF",
+        "CJSON_OVERRIDE_BUILD_SHARED_LIBS": "ON",
+        "CJSON_BUILD_SHARED_LIBS": "OFF",
+        "BUILD_SHARED_AND_STATIC_LIBS": "OFF",
+        "ENABLE_TARGET_EXPORT": "OFF",
+        "ENABLE_CJSON_VERSION_SO": "OFF"
+      }
+    },
+    {
+      "name": "from-package-source",
+      "binaryDir": "${sourceDir}/build",
+      "cacheVariables": {
+        "NOPORTS_MBEDTLS_PATH": "deps/mbedtls-src",
+        "NOPORTS_CJSON_PATH": "deps/cjson-src",
+        "NOPORTS_ATSDK_PATH": "deps/atsdk-src"
+        "BUILD_SHARED_LIBS": "OFF",
+        "CMAKE_BUILD_TYPE": "Release",
+        "CMAKE_C_FLAGS": "-std=c99 -Wno-error -Wextra -Werror-implicit-function-declaration -Wno-calloc-transposed-args",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/build/release-static/tmp-install-dir",
+        "CMAKE_EXPORT_COMPILE_COMMANDS": "OFF",
+        "ATSDK_BUILD_TESTS": "OFF",
+        "ATSDK_MEMCHECK": "OFF",
+        "ENABLE_TESTING": "OFF",
+        "ENABLE_PROGRAMS": "OFF",
+        "ENABLE_CJSON_TEST": "OFF",
+        "CJSON_OVERRIDE_BUILD_SHARED_LIBS": "ON",
+        "CJSON_BUILD_SHARED_LIBS": "OFF",
+        "BUILD_SHARED_AND_STATIC_LIBS": "OFF",
+        "ENABLE_TARGET_EXPORT": "OFF",
+        "ENABLE_CJSON_VERSION_SO": "OFF"
+      }
+    },
+    {
+      "name": "openwrt",
```

Comment:
new openwrt build, which can be used to generate the binaries from the source-package we push to the github release.